src/Application.php

```diff
@@ -107,6 +108,24 @@ private function registerCommands()
             $logger->writeln($this->trans('application.site.errors.settings'));
         }
 
+        $chainCommands = $this->getConfig()->get(
```

Comment:
can you remove from 111 to 115 the loading commands will not be reading a config file.
I will write the implementation